### PR TITLE
Dropped Python 3.6/Django 2.2 and 3.1 support.

### PR DESCRIPTION
**Pull request by claudep**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
+        python-version: [3.7, 3.8, 3.9, '3.10']
```
Comment (uri-rodberg) [inline .github/workflows/test.yml:10]: @claudep Maybe you can change it to `['3.7', '3.8', '3.9', '3.10']`?

**Review by uri-rodberg — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (claudep) [reply .github/workflows/test.yml]: Good suggestion.

**Review by uri-rodberg — CHANGES_REQUESTED** [review]
I'm sorry I didn't see it before, but you should also update this line in `README.rst`:

`Support for Django 2.2, 3.1, 3.2, 4.0 and 4.1 following the Django supported versions policy`

**Comment by claudep** [comment]
Good catch, updated.

**Review by uri-rodberg — APPROVED** [review]

**Comment by uri-rodberg** [comment]
@claudep I approve this PR.

**Pull request merged**